src/tutorial/microbe_stage/CheckTheHelpMenu.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Tutorial
+{
+    using System;
+    using Newtonsoft.Json;
+
+    public class CheckTheHelpMenu : TutorialPhase
+    {
+        public const string TUTORIAL_NAME = "CheckTheHelpMenu";
+
+        private const int TriggersOnNthSwimmingSession = 3;
```

Comment:
Might want to rename this TriggersOnNthEditorSession since that's what this class counts

Reply:
But this is the tutorial for the microbe stage? So I renamed this variable when copy-pasting.